proto/ethermint/crypto/v1alpha1/ethsecp256k1/keys.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+package ethermint.crypto.v1alpha1.ethsecp256k1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/cosmos/ethermint/crypto/ethsecp256k1";
+
+// PubKey defines a type alias for an ecdsa.PublicKey that implements
+// Tendermint's PubKey interface. It represents the 33-byte compressed public
+// key format.
+message PubKey {
+  option (gogoproto.goproto_stringer) = false;
```

Comment:
what does this option do?

Reply:
it allows you to create your custom `String() string` function instead of the default one. This is to add the ethereum address when printing the account